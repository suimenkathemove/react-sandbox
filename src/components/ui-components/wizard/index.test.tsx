import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import { Wizard } from './';

const setup = () => render(<Wizard {...defaultProps} />);

describe('Wizard', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
