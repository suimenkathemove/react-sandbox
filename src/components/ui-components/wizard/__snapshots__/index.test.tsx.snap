// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wizard render 1`] = `
<DocumentFragment>
  <div>
    <progress
      class="sc-bdvvtL jYuhrR"
      max="23"
      value="0"
    />
    <div
      style="display: flex; justify-content: space-between;"
    >
      <button
        style="visibility: hidden;"
      >
        prev
      </button>
      <div
        style="width: 200px;"
      >
        <div>
          <p>
            1
          </p>
          <input
            style="border: 1px solid black;"
            value=""
          />
        </div>
      </div>
      <button
        disabled=""
      >
        next
      </button>
    </div>
  </div>
</DocumentFragment>
`;
